src/databases/Blueprint/avtBlueprintFileFormat.C

```diff
@@ -847,6 +850,12 @@ avtBlueprintFileFormat::AddBlueprintMeshAndFieldMetadata(avtDatabaseMetaData *md
                 // treat the mesh as an mfem mesh, even if it lacks a basis func
 
                 m_mfem_mesh_map[var_topo_name] = true;
+
+                std::string basis = n_field["basis"].as_string();
+                if (basis.find("H1_") == std::string::npos && m_new_refine)
```

Comment:
Are all `H1` bases in MFEM zone centered? Does the `H1_` always occur at the *beginning* of the string? Must it always be capitalized?

Reply:
1) H1 bases are node centered. The if stmt checks if "H1" does *not* appear in the `basis` string. Cyrus and I both had the same confusion looking at this line and at a similar one in the data adaptor.

2) I'm not sure where "H1_" occurs in the string, although my best guess is that it is at the very beginning. This logic should catch it wherever it ends up in the string.

3) It should always be capitalized.

At Cyrus's direction, I copied this method of determining if the basis was L2 or H1 from code within Ascent. There may be a better way to do this. I've added some comments to clarify.